src/Geocoding/CacheEncoder/RichEncoder.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Geocoding\CacheEncoder;
+
+use Geocoder\Location;
+
+class RichEncoder implements CacheEncoder
+{
+    private const POSTFIX = '_enriched';
+
+    public function encode(Location $location): array
+    {
```

Comment:
Since this is for the duplicate places it might be a good idea to check with Frederic how they will get the data.
I doubt the data-team can access the cache. So maybe a logfile or database table?